index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for the RDFJS specification 4.0
+// Project: https://github.com/rdfjs/representation-task-force
+// Definitions by: Ruben Taelman <https://github.com/rubensworks>
+//                 Laurens Rietveld <https://github.com/LaurensRietveld>
+//                 Tomasz Pluskiewicz <https://github.com/tpluscode>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
Don't know if we want this header from DT

Reply:
I'd say remove it